### ISSUE TITLE
Add Leader Board

### ISSUE THREAD

**Issue by akshat-jn-crypto** [akshat-jn-crypto/Wordle-clone#2]
That is ask name of particpant the give 10 point for each correct score

**Comment by nalin243** [comment]
How do you want it implemented? I could add a leader board on the left or the right side of the page using a table and storage will be taken care of by localStorage. How do I ask the participant name? Will a simple prompt do or should another page be added for entering name? I could also just ask the username on the same page just above or below the leaderboard.

**Comment by akshat-jn-crypto** [comment]
> How do you want it implemented? I could add a leader board on the left or the right side of the page using a table and storage will be taken care of by localStorage. How do I ask the participant name? Will a simple prompt do or should another page be added for entering name? I could also just ask the username on the same page just above or below the leaderboard.  @akshat-jn-crypto 

Yes leader board on the left side and a prompt to ask participants name would be great.

**Comment by nalin243** [comment]
Could you assign this to me? @akshat-jn-crypto

**Comment by akshat-jn-crypto** [comment]
Attach Screenshot of the changes in the pull request.
